configs/processors/arxiv_hybrid_v2.yaml

```
@@ -0,0 +1,134 @@
+
+# Multiprocessing Configuration
+multiprocessing:
+  # Docling extraction workers (CPU-bound)
+  extraction_workers: 16     # Start conservative, can override with CLI
+  extraction_memory_gb: 64  # Total RAM allocation for extraction workers
+  memory_per_worker_gb: 4   # Expected memory per worker (for monitoring)
+  
+  # GPU embedding workers (usually 1 per GPU)
+  gpu_workers: 4  # One per GPU
+  gpu_devices: [0, 1]  # RTX A6000 devices
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)